format/FormatCBFMultiTile.py

```diff
@@ -126,12 +126,20 @@ def _detector(self):
             try:
                 cbf.find_category(b"array_intensities")
                 cbf.find_column(b"undefined_value")
+                cbf.select_row(i)
                 underload = cbf.get_doublevalue()
-                overload = cbf.get_overload(0)
+                overload = cbf.get_overload(i)
                 trusted_range = (underload, overload)
             except Exception:
                 trusted_range = (0.0, 0.0)
 
+            try:
+                cbf.find_column(b"gain")
+                cbf.select_row(i)
+                gain = cbf.get_doublevalue()
+            except Exception:
```

Comment:
This introduces 3 `except Exception:`. Are there more specific exceptions we could use?

Reply:
No, but I've brought in a string check

Reply:
:+1: thanks